packages/preact/src/index.ts

```diff
@@ -172,13 +179,21 @@ Object.defineProperties(Signal.prototype, {
 /** Inject low-level property/attribute bindings for Signals into Preact's diff */
 hook(OptionsTypes.DIFF, (old, vnode) => {
 	if (typeof vnode.type === "string") {
+		const oldSignalProps = vnode.__np;
 		let signalProps: Record<string, any> | undefined;
 
 		let props = vnode.props;
 		for (let i in props) {
 			if (i === "children") continue;
 
 			let value = props[i];
+			if (
+				value &&
+				typeof value === "object" &&
+				Object.getPrototypeOf(value) === jsxBind
```

Comment:
Just using .prototype will be lower bytes and more performant I think, that being said not too fuzzed can look at this later. Will do one final pass over the weekend and get this merged

Reply:
that wouldn't be equivalent. it's either getPrototypeOf or `.__proto__`

Reply:
@developit The prototype needs to inherit from `Signal` (see line below jsxBind declaration) for the `SignalValue` component to be used (`jsxBind(…)` expression as JSX child).

personally, I don't see an issue with using `__proto__` as I did originally, but I digress